### DISCUSSION TITLE
宝塔面板的在线编辑器中，无法准确选中字符

### DISCUSSION THREAD

**Discussion by popokcn** [F9y4ng/GreasyFork-Scripts#390] [问答专区(Question / Answer)]
![image](https://github.com/user-attachments/assets/400642c2-eb10-4337-9db5-1d96492f1b72)

如图所示，我鼠标选择的是<title>SKU查询</title>，但是实际上复制得到的是  <title>SKU查询<

就是说选中内容往前便宜了几个字符，和看到的选中部分不同。

配置如下，基本都是默认配置
![PixPin_2024-11-05_12-47-53](https://github.com/user-attachments/assets/e9a7623f-e695-424a-af27-d81f57603b88)
![PixPin_2024-11-05_12-48-04](https://github.com/user-attachments/assets/e2fe2288-6e87-42b7-a063-73aebb4bab64)

**Comment by F9y4ng** [comment]
开启自定义等宽字体再试试。

如果依然存在问题，那只能排除渲染来解决了。部分编辑器对字宽都有限定，即使等宽字体也有不同的字宽，所以要么找到编辑器指定字宽的等宽字体来使用，或者排除渲染。

**Reply by popokcn** [reply]
自定义等宽字体选项在哪里，没找到。你的意思是选一个等宽的字体使用吗？

我排除渲染倒是没问题了

**Reply by F9y4ng** [reply]
自行查阅 [Wiki](https://github.com/F9y4ng/GreasyFork-Scripts/wiki)

**Comment by popokcn** [comment]
好了，看到 ，谢谢。

顺便分享一个宝塔后天编辑器的选择器：#ace_editor

至于域名应该是不确定吧，不会域名可不可以模糊匹配

[image: PixPin_2024-11-05_16-02-41.png]

F9y4ng ***@***.***> 于2024年11月5日周二 15:27写道：

**Reply by F9y4ng** [reply]
等宽字体默认规则中已包含该编辑器，无需重复添加。
